samples/media/video/src/main/java/com/example/platform/media/video/TransformerVideoComposition.kt

```diff
@@ -157,28 +160,51 @@ class TransformerVideoComposition : Fragment() {
     private fun createComposition(): Composition {
         val video1 = EditedMediaItem.Builder(
             // apply effects only on the first item
-            MediaItem.fromUri(URI_ITEM1))
+            MediaItem.fromUri(URI_VIDEO1),
+        )
             .setEffects(getSelectedEffects())
             .build()
         val video2 = EditedMediaItem.Builder(
-            MediaItem.fromUri(URI_ITEM2))
+            MediaItem.fromUri(URI_VIDEO2),
+        )
+            .build()
+        val image = EditedMediaItem.Builder(
+            MediaItem.fromUri(URI_IMAGE),
+        )
+            .setDurationUs(3_000_000) // Show the image for 3 seconds in the composition
+            .setFrameRate(30)
             .build()
         val compositionSequences = ArrayList<EditedMediaItemSequence>()
-        val videoSequence = EditedMediaItemSequence(ImmutableList.of(video1, video2))
-        compositionSequences.add(videoSequence)
+
+        val videoSequence = arrayListOf<EditedMediaItem>()
+
+        if (binding.imageChip.isChecked) {
+            // Add image as first item in video sequence
```

Comment:
Lines 171 through 176 can go in here since image is optional and we should not need to create EditedMediaItem for an image every time